popgen/stationarity.go

```diff
@@ -176,3 +176,34 @@ func AfsLikelihood(afs Afs, alpha []float64, binomMap [][]float64, integralError
 	}
 	return answer
 }
+
+//AfsLikelihoodFixedAlpha calculates the likelihood of observing a particular frequency spectrum for a given alpha, a selection parameter.
+//This represents the special case where every segregating site has the same value for selection, which enables faster, more simplified calculation.
+func AfsLikelihoodFixedAlpha(afs Afs, alpha float64, binomMap [][]float64, integralError float64) float64 {
```

Comment:
Is the idea with these 3 new functions that since alpha is fixed we can do lots of caching?

Reply:
Yes! Especially for sets like the UCEs, where half of the variants have a dAF of 1/n, we can save a substantial amount of runtime by caching probabilities.